src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -254,9 +254,8 @@ class ColumnDefinitionSegment(BaseSegment):
                 OneOf(
                     Sequence(
                         "CURRENT_TIMESTAMP",
-                        Sequence(
-                            Bracketed(Ref("NumericLiteralSegment")),
-                            optional=True,
+                        Bracketed(
+                            Ref("NumericLiteralSegment", optional=True), optional=True
```

Comment:
How does this fix the issue? FWIW, the old code looks wacky to me.

Reply:
There's two changes in this:
1. I added optional to `Ref("NumericLiteralSegment", optional=True)` - that fixes the issue.
2. I removed an unnecessary `Sequence` as it was a `Sequence` of one item (`Bracketed`) and you only need to use `Sequence` when more than one item. Just a bit of clean up as was unnecessary IMHO. Unrelated to this fix.

Reply:
I agree, `Sequence` seemed wacky here.